python/py-treq/Portfile

```diff
@@ -20,11 +20,11 @@ distname            ${python.rootname}-${version}
 
 license             MIT
 
-checksums           sha256  91e09ff6b524cc90aa5e934b909c8d0d1a9d36ebd618b6c38e37b17013e69f48 \
-                    rmd160  0960e004dc9769df3228915bba439acc41162294 \
-                    size    59156
+checksums           sha256  68fee7d24c94b7f1432ad8077e22d0d15f957bf6fcf9cb771ff4c6acc7e2bc84 \
+                    rmd160  fee959996e5eb2d999a3069c0c618129848d0f06 \
+                    size    59116
 
-python.versions     37 38
+python.versions     38
```

Comment:
Looks like only py-buildbot depends on py-treq, and py-buildbot is only available as py38, so this removal of py37 here is fine.